_episodes/05-join.md

```diff
@@ -482,7 +482,7 @@ WHERE 1=CONTAINS(
 We run the query again.
 
 ~~~
-job = Gaia.launch_job_async(query=query)
+cone_job = Gaia.launch_job_async(query=cone_query)
```

Comment:
Do you think this should have a different name since we use `cone_job` in episode 2?